src/common/api.test.ts

```diff
@@ -1,32 +0,0 @@
-import * as dns from '../dns/index'
-import { did } from './api'
-
-jest.mock('../dns')
```

Comment:
This might be the most controversial thing in this PR: I'm deleting this test, because I didn't figure out mocking yet. It's [supposedly possible](https://sinonjs.org/how-to/stub-dependency/), but I just haven't looked into it much.

I'm not sure whether introducing a mocking library to keep this test alive would be worth it, though.

I'm open to other's opinions, though!